src/vsr.zig

```diff
@@ -26,6 +26,9 @@ pub const lsm = .{
     .forest = @import("lsm/forest.zig"),
     .posted_groove = @import("lsm/posted_groove.zig"),
 };
+pub const testing = .{
+    .cluster = @import("test/cluster.zig"),
```

Comment:
Maybe rename the folder to `testing` as well? To me at least, there's some logic in `testing` being infra for testing, and `test` being actual tests. 

(can't `pub const test` because that's a keyword)

Reply:
Agreed, it also matches better with `std`'s "testing" module, and we have a state machine named "testing" too.

Reply:
let me do that here

Reply:
Done!